test_regress/t/t_split_var_1_bad.out

```diff
@@ -75,4 +71,16 @@
                                               : ... In instance t.i_sub1
    66 |       rd_data = rd_tmp[{3'b0, addr[0]}+:8];
       |                       ^
+%Error: t/t_split_var_1_bad.v:78:11: Illegal assignment of constant to unpacked array
```

Comment:
I think this first error is actually better than the old one. Though the other two are a bit confusing, perhaps they could use rewording. We'll let Wilson comment on this.